src/qml/controls/TextButton.qml

```diff
@@ -9,14 +9,16 @@ Button {
     id: root
     property int textSize: 18
     property string textColor: Theme.color.neutral9
+    property bool bold: true
```

Comment:
you could make this it's own commit, also maybe switch the logic from defining `rightalign` to `center`?:

qml: make text boldness and alignment customizable in TextButton control
```diff

diff --git a/src/qml/controls/TextButton.qml b/src/qml/controls/TextButton.qml
index 2dce44670..4f508970d 100644
--- a/src/qml/controls/TextButton.qml
+++ b/src/qml/controls/TextButton.qml
@@ -10,13 +10,15 @@ Button {
     property int textSize: 18
     property string textColor: Theme.color.neutral9
     font.family: "Inter"
-    font.styleName: "Semi Bold"
+    font.styleName: bold ? "Semi Bold" : "Regular"
     font.pixelSize: root.textSize
+    property bool bold: true
+    property bool center: true
     contentItem: Text {
         text: root.text
         font: root.font
         color: root.textColor
-        horizontalAlignment: Text.AlignHCenter
+        horizontalAlignment: center ? Text.AlignHCenter : Text.AlignRight
         verticalAlignment: Text.AlignVCenter
     }
     background: null
```